app/ui-react/syndesis/src/modules/data/pages/VirtualizationViewsPage.tsx

```diff
@@ -187,82 +187,88 @@ export class VirtualizationViewsPage extends React.Component<
                                           helpers.isSortAscending
                                         );
                                         return (
-                                          <>
-                                            <PageSection
-                                              variant={'light'}
-                                              noPadding={true}
-                                            >
-                                              <VirtualizationNavBar
-                                                virtualization={virtualization}
+                                          <WithLoader
+                                            error={error}
+                                            loading={!hasData}
+                                            loaderChildren={
+                                              <ViewListSkeleton
```

Comment:
Did you figure out that black bar?  Here I think one option is you could put a `PageSection` around this *or* maybe better use the `PageLoader` [component](https://github.com/syndesisio/syndesis-react/blob/master/app/ui-react/packages/ui/src/Layout/PageLoader.tsx) instead that @riccardo-forina  had mentioned.  The PageLoader shows a spinner though.  Other option could be to borrow [this guy](https://github.com/syndesisio/syndesis-react/blob/master/app/ui-react/packages/ui/src/Integration/IntegrationsListSkeleton.tsx) it's used on the main integration list page and we also borrowed it for the oauth settings page.

Reply:
Yeah, this is one of the pages with that issue yet.  I'll look at fixing in another PR.  Thx!